batch/vep/run_vep.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script is intended to be used in the Docker image built for VEP.
+# Note that except the single input and output files, all other arguments are
+# passed through environment variables.
+#
+# The only environment variable that has to be set is (others are optional):
+#
+# VEP_CACHE: The path of the VEP cache which is a single .tar.gz file.
+#
+# The first argument is the input file (might be a VCF or a compressed VCF) and
+# the second is the output file which is always a VCF file.
+#
+# For the full list of supported environment variables and their documentation
+# check README.md.
+# Capital letter variables refer to environment variables that can be set from
+# outside. Internal variables have small letters.
+
+set -euo pipefail
+
+if [[ $# -ne 2 ]]; then
+  echo "Usage: $0 input_file output_file"
+  exit 1
+fi
+
+readonly species="${SPECIES:-homo_sapiens}"
+readonly assembly="${GENOME_ASSEMBLY:-GRCh38}"
+readonly fork_opt="--fork ${NUM_FORKS:-1}"
+readonly other_vep_opts="${OTHER_VEP_OPTS:---everything}"
+readonly annotation_field_name="${VCF_INFO_FILED:-CSQ_VT}"
```

Comment:
Given that we're now outside of VT context, please update the default to just CSQ

Reply:
Done.